### DISCUSSION TITLE
building problems

### DISCUSSION THREAD

**Discussion by 394781865** [apache/incubator-mxnet#16775] [Q&A] [answered]
Makefile:559: recipe for target 'build/src/operator/numpy/random/np_uniform_op_gpu.o' failed
make: *** [build/src/operator/numpy/random/np_uniform_op_gpu.o] Error 1

what is the problem?my numpy version is 1.16.

**Comment by reminisce** [comment, marked as answer]
What's the error message? Did you start the build from a clean state?

**Comment by samskalicky** [comment]
@zachgk assign @reminisce

**Comment by haozeze** [comment]
@394781865  have you sovled this problem?    I got the same one
